js/main.js

```diff
@@ -202,6 +203,11 @@ Site.Layout = {
 
     if ($('body').hasClass('single-collection')) {
       _this.convertScroll();
+      
+      $('img.single-collection-item').one('load',function() {
```

Comment:
Why this specific binding rather than checking for the existence of these items? If need is for images to be loaded for correct widths better to use imagesLoaded plugin on the entire container as that will wait for all not just the first.